app/views/course/groups/_group.html.slim

```diff
@@ -1,8 +1,10 @@
 = content_tag_for(:tr, group) do
   th = link_to format_inline_text(group.name), course_group_path(current_course, group)
-  td = group.course_users.without_phantom_users.count
-  td = group.course_users.phantom.count
-  td = group.course_users.count
+  - normal_user_count = group.group_users.count { |group_user| !group_user.course_user.phantom? }
+  - all_user_count = group.group_users.size
+  td = normal_user_count
+  td = all_user_count - normal_user_count
```

Comment:
Can add a comment saying this is the phantom count? It's possible to infer from the code but not immediately explicit.